e2e/playwright/viewer/src/tests/viewer-protected.spec.ts

```diff
@@ -29,15 +29,15 @@ test.use({ storageState: getUserState('hruser') });
 test.describe('viewer file', () => {
   const apiClientFactory = new ApiClientFactory();
   const randomFolderName = `playwright-folder-${Utils.random()}`;
-  const randomDocxName = `${TEST_FILES.DOCX_PROTECTED.name}-${Utils.random()}`;
+  const randomDocxName = `${TEST_FILES.PDF_PROTECTED.name}-${Utils.random()}`;
   let folderId: string;
   let fileDocxId: string;
 
   test.beforeAll(async ({ fileAction, shareAction, favoritesPageAction: favoritesPageAction }) => {
     await apiClientFactory.setUpAcaBackend('hruser');
     const node = await apiClientFactory.nodes.createNode('-my-', { name: randomFolderName, nodeType: 'cm:folder', relativePath: '/' });
     folderId = node.entry.id;
-    const fileDoc = await fileAction.uploadFile(TEST_FILES.DOCX_PROTECTED.path, randomDocxName, folderId);
+    const fileDoc = await fileAction.uploadFile(TEST_FILES.PDF_PROTECTED.path, randomDocxName, folderId);
```

Comment:
Why we are changing the doc format?

Reply:
previously wrong file name was given as  docx but file type was PDF, so corrected the file name.